dt-groups/groups.js

```diff
@@ -204,14 +207,15 @@ jQuery(document).ready(function($) {
     }
   })
   $(document).on("click", ".make-leader", function () {
+    $(this).children('i').attr('class', 'small')
+    let spinner = `<img src="/wp-content/themes/disciple-tools-theme/dt-core/admin/img/spinner.svg" width="15px">`
+    $(this).append(spinner)
     let id = $(this).data('id')
     let remove = false
     let existingLeaderIcon = $(`.member-row[data-id="${id}"] .leader`)
     if( window.lodash.find( post.leaders || [], {ID: id}) || existingLeaderIcon.length !== 0){
       remove = true
       existingLeaderIcon.remove()
```

Comment:
@prykon  Remove `existingLeaderIcon.remove()` and the spinner will work both ways: